doc/lua_api.txt

```diff
@@ -3304,7 +3304,11 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `set_wielded_item(item)`: replaces the wielded item, returns `true` if successful
 * `set_armor_groups({group1=rating, group2=rating, ...})`
 * `get_armor_groups()`: returns a table with the armor group ratings
-* `set_animation({x=1,y=1}, frame_speed=15, frame_blend=0, frame_loop=true)`
+* `set_animation(frame_range, frame_speed, frame_blend, frame_loop)`
+  * `frame_range`: table {x=num, y=num}, default: `{x=1.0, y=1.0}`
```

Comment:
'1.0' implies floats can be used, these need to specify 'integers'.

Reply:
they are floats, see https://github.com/minetest/minetest/blob/master/src/script/lua_api/l_object.cpp#L479 (2d float vector)

Reply:
Maybe it implies float/double because it IS such a number? `frame_range` is `v2f`

Reply:
v2f may only be used because we don't have a v2int, need to look at the code to see.

Reply:
I'll check.